src/js/stores/library.js

```diff
@@ -270,6 +447,31 @@ define(function (require, exports, module) {
                 }
             }
         },
+        
+        /**
+         * Find element in all libraries by its ID.
+         *
+         * @private
+         * @param {string} elementID
+         * @return {?AdobeLibraryElement}
+         */
+        _findElementByID: function (elementID) {
+            var libraries = this._libraryCollection.libraries;
+            
+            for (var i = 0; i < libraries.length; i++) {
```

Comment:
if you pass in libraryID as well, you can use `AdobeLibraryCollection.getLibraryByID` and `AdobeLibraryComposite.getElementByID`

Although, what's the best to do is, instead of the ID, store the element reference, and use `CCLibraries.ResolveElementReference` to directly get the element.